src/site/content/en/fast/lazy-loading/index.md

```diff
@@ -0,0 +1,119 @@
+</figure>
+
+You've probably already seen lazy-loading in action, and it goes something like
+this:
+
+- You arrive at a page, and begin to scroll as you read content.
+- At some point, you scroll a placeholder image into the viewport.
+- The placeholder image is suddenly replaced by the final image.
+
+An example of image lazy-loading can be found on the popular publishing platform
+[Medium](https://medium.com/), which loads lightweight placeholder images at
+page load, and replaces them with lazily-loaded images as they're scrolled into
+the viewport.
+
+<figure class="w-figure">
+  <img src="./lazy-loading-example.jpg" alt="A screenshot of the website
+Medium in the browsing, demonstrating lazy-loading in action. The blurry
+placeholder is on the left, and the loaded resource is on the right.">
+  <figcaption class="w-figcaption">An example of image lazy-loading in action. A
+placeholder image is loaded at page load (left), and when scrolled into the
+viewport, the final image loads at the time of need.</figcaption>
+</figure>
+
+If you're unfamiliar with lazy-loading, you might be wondering just how useful
+the technique is, and what its benefits are. Read on to find out!
+
+## Why lazy-load images or video instead of just _loading_ them? {: #why }
+
+Because it's possible you're loading stuff the user may never see. This is
+problematic for a couple reasons:
+
+- It wastes data. On unmetered connections, this isn't the worst thing that could
+happen (although you could be using that precious bandwidth for downloading
+other resources that are indeed going to be seen by the user). On limited data
+plans, however, loading stuff the user never sees could effectively be a waste
+of their money.
+- It wastes processing time, battery, and other system resources. After a media
+resource is downloaded, the browser must decode it and render its content in the
+viewport.
+
+When we lazy-load images and video, we reduce initial page load time, initial
+page weight, and system resource usage, all of which have positive impacts on
+performance.
+
+## Implementing lazy-loading {: #implementing }
+
+There are a number of ways to implement lazy-loading.
+Your choice of solution must take into account the browsers you support,
+and also what you are trying to lazy-load.
+
+Modern browsers implement native lazy-loading, which can be enabled using the `loading` attribute on images and iframes.
+To provide compatibility with older browsers
+or to perform lazy-loading on elements without native lazy-loading
+you can implement a solution with your own JavaScript.
+There are also a number of existing libraries to help you to do this.
+See the guides on this site for full details of all of these approaches:
+
+- [Lazy-loading images](/lazy-loading-images/)
+- [Lazy-loading-video](/lazy-loading-video/)
+
+Also, we have compiled a list of [potential issues with lazy-loading](/lazy-loading-problems),
+and things to watch out for in your implementation.
```

Comment:
I suggested renaming that post to "best practices" so this should be rephrased accordingly. (And the URL should be updated.)